tests/v3/test_store/test_remote.py

```diff
@@ -41,7 +42,7 @@ def get_boto3_client():
     return session.create_client("s3", endpoint_url=endpoint_uri)
 
 
-@pytest.fixture()
+@pytest.fixture(autouse=True)
```

Comment:
Double check: `autouse` scapes to this module alone? We don't want moto starting for all tests everywhere. I am happy to explicitly require `s3` as an argument to tests, since often we want the instance too.

Reply:
good question, I have no idea what scope `autouse` has, and I will read into it. Can you explain more about why we would want the `s3` instance itself in a test? My impression was that the endpoint_uri + bucketname should be sufficient for all our tests.

Reply:
I set the scope of the `s3` fixture to `session`